project/source/collision_events/WasdController.h

```diff
@@ -3,6 +3,8 @@
 #include "Ecs.h"
 #include "Input.h"
 
+#include <iostream>
```

Comment:
remove

Reply:
Oh, I see you're using it below. Not sure if you intended to keep the prints around.